presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -75,18 +75,23 @@ contents:
    access-control.name=file
    security.config-file=etc/rules.json
 
-The config file is specified in JSON format.
+The config file is specified in JSON format. It contains rules that define
+which users have access to which resources:
 
-* It contains the rules defining which catalog can be accessed by which user (see Catalog Rules below).
-* The query rules specifying which queries can be managed by which user (see Query Rules below).
-* The impersonation rules specify which user impersonations are allowed (see Impersonation Rules below).
-* The principal rules specifying what principals can identify as what users (see Principal Rules below).
-* The system information rules specifying what users can access system management information (see System Information Rules below).
+* :ref:`Catalogs <catalog_rules>`
```

Comment:
We should avoid adding refs for links within the same document. They duplicate the built-in anchors for section titles, and the names are global within Sphinx, so care must be taken when naming. Instead, you can link to the built-in section anchors:
```
* `Catalogs <#catalog-rules>`__